src/sentry/lang/native/plugin.py

```diff
@@ -421,11 +451,14 @@ def process_frame(self, frame):
             else:
                 logger.debug('Failed to symbolicate with native backend',
                              exc_info=True)
-        else:
+            return None, [raw_frame], errors
+
+        for sfrm in symbolicated_frames:
             symbol = sfrm.get('symbol_name') or \
-                new_frame.get('function') or '<unknown>'
+                frame.get('function') or '<unknown>'
```

Comment:
One thing I noticed on the PR before ... maybe we should put the `<unkown>' string in a global constant